### PR TITLE
Fix Kernel.function_exported? call

### PR DESCRIPTION
**Pull request by indocomsoft**
I've added:

- [ ] USAGE.md docs if applicable
- [X] CHANGELOG.md

### CODE REVIEW

**Comment by indocomsoft** [comment]
Fixes #204

**Comment by indocomsoft** [comment]
This will only happen when `Faker.mlocale()` evaluates to `nil`. For example, during non-interactive sessions when the `Application.get_env(:faker, :locale)` evaluates to `nil`. I guess this is why the doctest didn't catch it

File: lib/faker.ex [hunk 1]
```diff
@@ -126,11 +126,12 @@ defmodule Faker do
     quote do
       def unquote(function)() do
         module = Module.concat(unquote(__CALLER__.module), Faker.mlocale())
+        module_enus = Module.concat(unquote(__CALLER__.module), Faker.mlocale())
         cond do
           function_exported?(module, unquote(function), 0) ->
             apply(module, unquote(function), [])
-          function_exported?(module, unquote(function), EnUs) ->
-            apply(Module.concat(unquote(__CALLER__.module), EnUs), unquote(function), [])
+          function_exported?(module_enus, unquote(function), 0) ->
```
Comment (igas) [inline lib/faker.ex]: I don't see the difference between this and the previous cond branch. I don't think it fixes the problem, just removes the feature. The idea of this one is to apply the default country.

Comment (indocomsoft) [reply lib/faker.ex]: The third parameter to `Kernel.function_exported?` is the arity of the function. `EnUs` is not arity. Instead, the module name should be concatenated with `EnUs`, and then check whether the function with arity `0` is exported.

Comment (indocomsoft) [reply lib/faker.ex]: Note that it's checking `module_enus` not `module`

Comment (indocomsoft) [reply lib/faker.ex]: Oops, my bad. I should be concatenating it with `EnUs` not `Faker.mlocale`

Comment (indocomsoft) [reply lib/faker.ex]: Fix pushed!

**Comment by indocomsoft** [comment]
Bump 😄

**Comment by igas** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**